src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -313,6 +313,16 @@ export default {
 		useFullWidth: {
 			type: Boolean,
 			default: false
+		},
+		atcTrackingProps: {
```

Comment:
We should look to remove this and use the existing `add-to-basket` event that is already emitted